packages/core/src/credential/Credential.ts

```diff
@@ -399,25 +360,34 @@ export async function verify(
         'CType verification failed'
       )
   }
+}
 
-  if (challenge || credential.claimerSignature) {
-    const isSignatureCorrect = await verifySignature(credential, {
-      challenge,
-      didResolve,
-    })
-    if (!isSignatureCorrect)
-      throw new SDKErrors.CredentialUnverifiableError(
-        'Signature not verifiable'
-      )
-  } else if (!allowUnsigned) {
-    throw new SDKErrors.CredentialUnverifiableError(
-      'Signature required, but not provided'
-    )
-  }
+/**
+ * Verifies data structure, data integrity and the claimer's signature.
+ *
+ * Upon presentation of a credential, a verifier would call this [[verify]] function.
+ *
+ * @param presentation - The object to check.
+ * @param options - Additional parameter for more verification steps.
+ * @param options.ctype - CType which the included claim should be checked against.
+ * @param options.challenge -  The expected value of the challenge. Verification will fail in case of a mismatch.
+ * @param options.didResolve - The function used to resolve the claimer's identity. Defaults to [[resolve]].
+ */
+export async function verify(
```

Comment:
https://github.com/KILTprotocol/sdk-js/pull/618#pullrequestreview-1098752970

Guess this can only be decided by a spoon fight to the death.

Reply:
I think it is not a strong argument, since we rename a lot of other stuff, too. The whole credential is not even the old credential anymore.
Important is, that we don't break externally facing things and serializable data.